### ISSUE TITLE
CLI with config file "TypeError: Path must be a string. Received { colors: ... "

### ISSUE THREAD

**Issue by smozely** [tailwindcss/tailwindcss#147]
Hi there,

Feeling like this must be an environment thing, otherwise I couldn't be the only person getting this!

I'm running Node 8.9.0 / NPM 5.5.1 and whenever I try to specify the config file on the command line I get this error, note if I don't specify a config file it works fine.

Even tried it with a blank node project i.e.
```
npm init
npm install tailwindcss --save-dev
<created a tailwind.css file>
node_modules/.bin/tailwind build tailwind.css -c tailwind.js -o output.css
```

And I always get this error ... 
```
Building Tailwind!
path.js:28
    throw new TypeError('Path must be a string. Received ' + inspect(path));
    ^

TypeError: Path must be a string. Received { colors:
   { transparent: 'transparent',
     black: '#222b2f',
     'grey-darkest': '#364349',
     'grey-darker': '#596a73',
     'grey-dark': '#70818a',
     grey: '#9babb4',
     'grey-light': '#dae4e9',
     'grey-lighter': '#f3f7f9',
     'grey-lightest': '#fafcfc',
     white: '#ffffff',
     'red-darkest': '#420806',

... the rest of the tailwind config
```

Any ideas?

**Comment by giuliac1989** [comment]
I'm having the same issue

**Comment by m1guelpf** [comment]
Somewhat related to https://github.com/tailwindcss/discuss/issues/37

**Comment by adamwathan** [comment]
Ugh our mistake. I've pushed a fix and will tag a new release shortly.

**Comment by adamwathan** [comment]
Fixed in 0.1.5 👍
